app/(tabs)/reservation.tsx

```diff
@@ -0,0 +1,38 @@
+import { StyleSheet } from 'react-native'
+import { Text, View } from '@/components/Themed'
+import { Link } from 'expo-router'
+
+export default function Reservations() {
+    return (
+        <View style={styles.container}>
+            <Text style={styles.title}>Reservations</Text>
+            <View
+                style={styles.separator}
+                lightColor="#eee"
+                darkColor="rgba(255,255,255,0.1)"
```

Comment:
you are using this colors in more then one place, so maybe put them in Color.ts file